src/controllers/projectController.js

```diff
@@ -27,4 +28,22 @@ module.exports = {
       return res.status(500).json({ message: 'Erro ao cadastrar projeto' });
     }
   },
+  async editProject(req, res) {
+    const projectToCheck = await projectModel.verifyId(
+      'projects',
+      'project_id',
```

Comment:
Eu ia sugerir de você usar um Enum nesse caso, deixar essas strings soltar no código pode te atrapalhar pra caso você mude o nome das colunas. Deixando num enum você consegue rastrear onde que tá sendo usado, alterar só o enum e já aplicar a alteração pra codebase inteira.

Outra coisa também é que não dá pra saber o que essas strings dizem a respeito, se é uma coluna da tabela ou se é a própria tabela. Deixando isso debaixo dessa estrutura nomeada, fica fácil de entender. Vou deixar o exemplo de TS e JS:

No TypeScript:
```typescript
enum DatabaseEnum {
   PROJECTS = 'projects',
   PROJECT_ID = 'project_id'
}
```

No JavaScript:
```javascript
const DatabaseEnum = {
   PROJECTS: 'projects',
   PROJECT_ID: 'project_id'
}
```

```suggestion
      DatabaseEnum.PROJECTS,
      DatabaseEnum.PROJECT_ID,
```